common/src/main/java/com/wynntils/mc/utils/ItemUtils.java

```diff
@@ -185,6 +187,20 @@ public static void removeItemLore(List<Component> tooltip) {
         tooltip.removeAll(toRemove);
     }
 
+    /**
+     * Returns true if the passed item is a Wynncraft item (armor, weapon, accessory)
+     */
+    public static boolean isWynnItem(ItemStack itemStack) {
```

Comment:
these two new functions probably belong in WynnItemUtils, as does the original `isMythic` one - the minecraft package is meant for stuff that isn't specific to wynncraft

Reply:
WynnItemMatcher exists, we should use that.

Reply:
Fixed this.